test/cli.test.js

```diff
@@ -171,6 +246,22 @@ test('push command fails for invalid CSV file', async t => {
 
 // end of [Pushing invalid CSV file (irrespective of schema)]
 
+// QA tests [Pushing packaged invalid CSV file (irrespective of schema)]
+
+test.serial('push command succeeds with packaged invalid CSV', async t => {
+  const path_ = 'test/fixtures/test-data/packages/invalid-data'
+  const result = await runcli('push', path_)
+  const stdout = result.stdout.split('\n')
+  const hasPublishedMessage = stdout.find(item => item.includes('your data is published!'))
+  const hasURLtoShowcase = stdout.find(item => item.includes('https://datahub.io/test/basic-csv/v/'))
+  t.truthy(hasPublishedMessage)
```

Comment:
Why here is successful message, if we are testing invalid CSV file? Shouldn't pushing FAIL?

Reply:
@AcckiyGerman when a CSV is packaged we don't validate it against the schema (so we don't read it). You'd say that we could do it, but in that case what's the reason of having validation report on the showcase page? 
Another reason is that you'd like to do some processing with your original data using our pipelines system. E.g., you can replace some characters or set field types etc. As you know you can specify these steps in `flow.yaml` file.
So in general, this might be useful when you really want to keep your original CSV even if it's invalid but you want to derive something valid using our backend.

/cc @zelima

Reply:
agree